e2e/tests/datasets/export-to-notebook.spec.ts

```diff
@@ -77,8 +79,96 @@ describe('Create Dataset', () => {
 
     // Delete Dataset.
     await dataPage.openTab(TabLabelAlias.Data);
-    await dataPage.openTab(TabLabelAlias.Datasets);
+    await waitWhileLoading(page);
+
+    await dataPage.openTab(TabLabelAlias.Datasets, {waitPageChange: false});
+    await waitWhileLoading(page);
+
     await dataPage.deleteDataset(newDatasetName);
+  });
+
+  /**
+   * Create new Cohort thru Dataset Build page. Cohort built from demographics -> Ethnicity.
+   * Create new Dataset with Cohort, then export to notebook in R language.
+   * Delete Cohort, Dataset, and Notebook.
+   */
+  test('Export dataset to notebook in R programming language', async () => {
```

Comment:
nice, this is pretty easy to read through

Reply:
Thanks. I'll try do more like this in future.